images/ansible-operator/ansible_runner_http/ansible_runner_http/events.py

```
@@ -0,0 +1,45 @@
+import os
+import requests
+import requests_unixsocket
+import logging
+
+logger = logging.getLogger('ansible-runner')
+
+def send_request(url, data, headers={}, urlpath=None):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)